lib/output.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+let cli = require('heroku-cli-util')
+
+let Stream = function (url) {
```

Comment:
Can we test this promise generator in isolation, as well? It's elegantly-written, so I don't expect the tests will uncover much, but I'd like to be sure…